js/web/lib/wasm/proxy-worker/main.ts

```diff
@@ -3,15 +3,47 @@
 
 /// <reference lib="webworker" />
 
-import {OrtWasmMessage} from '../proxy-messages';
+//
+// * type hack for "HTMLImageElement"
+//
+// in typescript, the type of "HTMLImageElement" is defined in lib.dom.d.ts, which is conflict with lib.webworker.d.ts.
+// when we use webworker, the lib.webworker.d.ts will be used, which does not have HTMLImageElement defined.
```

Comment:
This description is confusing. If HTMLImageElement is not defined in lib.webworker.d.ts then why there is a conflict. I understand that there is a good reason to make this change but the description is not clear.

Reply:
Is it possible to add a library as suggested [here](https://stackoverflow.com/questions/42603783/missing-basic-dom-types-in-typescript-project)?